public/data/shivam-sharma7.json

```
@@ -0,0 +1,28 @@
+{
+"name": " Shivam Sharma ",
+  "type": "personal",
+  "bio": "Open Source Enthusiast | Fullstack Developer | DevOps Enthusiast",
+  "avatar": "https://github.com/shivam-sharma7.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/shivam-sharma7",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/meshivamsharma",
+      "icon": "twitter"
+    },
+    {
+      "name": "Visit my profile website",
```

Hashnode profile! It'll be way more better to write `Visit my Blog` or `Read my Articles`
```suggestion
      "name": "Visit my Blog",
```